app/containers/OrderList/index.js

```diff
@@ -42,7 +42,7 @@ class OrderList extends Component {
           />
```

Comment:
I get race condition here, the `this.props.isFetching` set to false before the data come into this component. So, the order list not being rendered. maybe @andy-shi88 could do the test too.
after place the order there is no order lists shown for me.

Reply:
if this issue persists in your environment, let's move `isfetching` state to local component and change the state accordingly in `componentWillReceiveProps`.

Reply:
will test it now. @kristoforusrp cc: @caudaganesh , I'll let you know the result.

Reply:
I just logged them out and it's actually `undefined` the whole `this.props`, I'll recheck just in case and come with a solution at best

Reply:
wait @kristoforusrp where do you get the problem? on my side it's actually working fine after I fix refetch my backend.

Reply:
I can't get ticket list after place the order, but let met me check it on my device directly.
I'll let you know ASAP.

Reply:
@kristoforusrp let me confirm, after placing order or after payment?

Reply:
just spot the problem @kristoforusrp , it's on condition checking in `TicketList/actions.js` on `fetchUserTicket` method, the condition checking for response is weird, and too much action dispatching action there I suppose, I'll need to check it ASAP but I left my laptop charger cable  there will fix this after getting new charger in the morning.